src/components/Steckbrief/FederalStateList.tsx

```diff
@@ -0,0 +1,112 @@
+export const FederalStateList: React.FC<Props> = ({ location }) => {
+  const stateCount = {};
+  const statePaths = {};
+  const addState = (state) => {
+    if (stateCount[state]) {
+      stateCount[state] += 1;
+    } else {
+      stateCount[state] = 1;
+    }
+  };
+  const { radschnellwege }: Queries.FederalStatesQuery = useStaticQuery(query);
+  radschnellwege.nodes.forEach(
+    ({ general: { from, to }, fromPath, toPath }) => {
+      addState(from.federalState);
+      statePaths[from.federalState] = fromPath;
+      if (stateCount[from.federalState] !== stateCount[to.federalState]) {
+        addState(to.federalState);
+        statePaths[to.federalState] = toPath;
+      }
+    }
+  );
+  return (
+    <div className="w-72">
+      <Listbox value={null} onChange={(setOpen) => setOpen(false)}>
+        {({ open }) => (
+          <>
+            <Listbox.Label className="block text-sm font-medium text-white">
+              Fitern nach Bundesland
+            </Listbox.Label>
+            <div className="relative mt-1">
+              <Listbox.Button className="relative w-full cursor-default rounded-md border border-gray-300 bg-white py-2 pl-3 pr-10 text-left shadow-sm focus:border-indigo-500 focus:outline-none focus:ring-1 focus:ring-indigo-500 sm:text-sm">
+                <span className="block truncate">{location}</span>
+                <span className="pointer-events-none absolute inset-y-0 right-0 flex items-center pr-2">
+                  <ChevronDownIcon
+                    className="h-5 w-5 text-gray-400"
+                    aria-hidden="true"
+                  />
+                </span>
+              </Listbox.Button>
+
+              <Transition
+                show={open}
+                as={Fragment}
+                leave="transition ease-in duration-100"
+                leaveFrom="opacity-100"
+                leaveTo="opacity-0"
+              >
+                <Listbox.Options className="absolute z-10 mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 text-base shadow-xl ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm">
+                  {Object.keys(stateCount)
+                    .sort()
+                    .map((state) => (
+                      <Listbox.Option
+                        key={state}
+                        className={({ active }) =>
+                          classNames(
+                            active
+                              ? 'bg-indigo-600 text-white'
+                              : 'text-gray-900',
+                            'relative cursor-default select-none py-2 pl-3 pr-9'
+                          )
+                        }
+                        value={state}
+                      >
+                        <Link to={statePaths[state]}>
+                          <span className="block truncate font-normal">
+                            {`${state} (${stateCount[state]})`}
```

Comment:
Das reicht so, richtig?

```suggestion
                            {state} ({stateCount[state]})
```